lib/trento_web/openapi/v2/schema/cluster.ex

```diff
@@ -0,0 +1,175 @@
+defmodule TrentoWeb.OpenApi.V2.Schema.Cluster do
+  @moduledoc false
+
+  require OpenApiSpex
+  require Trento.Domain.Enums.ClusterType, as: ClusterType
+
+  alias OpenApiSpex.Schema
+
+  alias TrentoWeb.OpenApi.V1.Schema.{Cluster, Provider, ResourceHealth, Tags}
+
+  defmodule AscsErsClusterNode do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "AscsErsClusterNode",
+      description: "ASCS/ERS Cluster Node",
+      type: :object,
+      properties: %{
+        attributes: %Schema{
+          type: :object,
+          description: "Node attributes"
+        },
+        filesystems: %Schema{
+          type: :array,
+          items: %Schema{type: :string},
+          description: "List of filesystems managed in this node"
+        },
+        name: %Schema{
+          type: :string,
+          description: "Node name"
+        },
+        resources: %Schema{
+          type: :array,
+          items: Cluster.ClusterResource,
+          description: "A list of Cluster resources"
+        },
+        roles: %Schema{
+          type: :array,
+          items: %Schema{type: :string, enum: ["ascs", "ers"]},
```

Comment:
This mean that we can have at most two roles, right?

Reply:
Yes, one node can have from 0 to 2 roles for each SAP system.
The normal scenario is when one node is only `ascs` and the other `ers`, but during a failover, one nodes takes both roles and the other none.
PD: Creating a tech-debt ticket to put the enum values in an `Enum` object, so we don't need to spread them all around the code